feature/qos/otg_tests/ingress_police_nhg/README.md

```diff
@@ -0,0 +1,216 @@
+              queuing-behavior: POLICE
+            exceed-action:
+              config:
+                drop: TRUE
+  classifers:
+    - classifer: “dest_A”
+      config:
+        name: “dest_A”
+      terms:
+        - term:   # repeated for address in destination A
+          config:
+            id: "match_1_dest_A"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_A"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+    - classifer: “dest_B”
+      config:
+        name: “dest_B”
+      terms:
+        - term:
+          config:
+            id: "match_1_dest_B"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_B"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+
+  input-policies:       # new OC subtree input-policies (/qos/input-policies)
+    - input-policy: "limit_group_A_2Gb"
+      config:
+        name: "limit_group_A_2Gb"
+        classifer: "dest_A"
+        scheduler-policy: "limit_2Gb"
+    - input-policy: "limit_dest_group_B_1Gb"
+      config:
+        name: "limit_dest_group_B_1Gb"
+        classifer: "dest_B"
+        scheduler-policy: "limit_1Gb"
+
+  interfaces:                  # this is repeated per subinterface (vlan)
+    - interface: "PortChannel1"
+      interface-ref:
+        config:
+          subinterface: 100
+    input:
+      config:
+        policies:  [            # new OC leaf-list (/qos/interfaces/interface/input/config/policies)
+          limit_dest_group_A_2Gb
+        ]
+  interfaces:                  # this is repeated per subinterface (vlan)
+    - interface: "PortChannel1"
+      interface-ref:
+        config:
+          subinterface: 200
+    input:
+      config:
+        policies:  [            # new OC leaf-list (/qos/interfaces/interface/input/config/policies)
+          limit_dest_group_B_1Gb
+        ]
+
+```
+
+### TE-18.3.1 Test traffic
+
+* Send traffic
+  * Send traffic from ATE port 1 to DUT for dest_A and is conforming to cir.
+  * Send traffic from ATE port 1 to DUT for to dest_B and is conforming to
+    cir.
+  * Validate packets are received by ATE port 2.
+    * Validate qos interface scheduler counters
+    * Validate afts next hop counters
+  * Validate outer packet ipv6 flow label assignment
+    * When the outer packet is IPv6, the flow-label should be inspected on the ATE.
+      * If the inner packet is IPv4, the outer IPv6 flow label should be computed based on the IPv4 5 tuple src,dst address and ports, plus protocol
+      * If the inner packet is IPv6, the inner flow label should be copied to the outer packet.
+  * Increase traffic on flow to dest_B to 2Gbps
+    * Validate that flow dest_B experiences ~50% packet loss (+/- 1%)
+
+# TE-18.3 Control maximum rate of traffic transmitted or received on an interface.
```

Comment:
Oh, I see you cut and paste the 18.2 test and added an 18.3 here.

Each readme is independent.  The 18.2 test above should be removed from this file.  

Additionally, this should be numbered `TE-18.4`   Also be more specific about the name of the test as there are many ways to rate limit traffic.  I suggest something like:

`TE-18.4 - Classify traffic on input using destination MAC and police using 1 rate, 2 color marker`